tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -279,11 +267,7 @@ def test_it_configures_frontend_grading(
         via_url,
         ModuleItemConfiguration,
     ):
-        pyramid_request.params = {
-            "resource_link_id": "TEST_RESOURCE_LINK_ID",
-            "tool_consumer_instance_guid": "TEST_TOOL_CONSUMER_INSTANCE_GUID",
-            **lti_outcome_params,
-        }
+        pyramid_request.params = lti_outcome_params
```

Comment:
All other calls to this in the file do:

`pyramid_request.params.update(lti_outcome_params)`

Should that be the same here?